api/schema/schema_validator.go

```diff
@@ -24,11 +24,12 @@ func NewSchemaValidator(t *testing.T, schemaName string, got interface{}) *Schem
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Platform compatibility: use "/" separators always for file://
+	dir = strings.Replace(dir, "\\", "/", -1)
```

Comment:
I agree that the no-op generalization is a little bit confusing, but I feel that the os.PathSeparator + a comment explaining what is going on is far preferable than depending on the `runtime` package.